### ISSUE TITLE
iOS - links in generated pdf are not clickable

### ISSUE THREAD

**Issue by tsafundzic** [penkzhou/plugin_html_to_pdf#5]
Affected: iOS
Android is working as expected.

Not all links are clickable in generated pdf:

For example: 
```
 <p><a href=\"https://www.google.com/\">GOOGLE</a> Link test</p> // Not clickable
 <p><a href=\"https://www.google.com/\">www.google.com</a></p> // Clickable

```
Looks like href attribute is ignored, and only <a> tags which shows the url are clickable in the generated pdf

**Comment by tsafundzic** [comment]
FIX: https://github.com/penkzhou/plugin_html_to_pdf/pull/6

**Comment by penkzhou** [comment]
Thanks again for your contribution @tsafundzic .It has been released as version `0.8.1`.
